src/twisted/mail/test/test_mailmail.py

```diff
@@ -73,3 +109,271 @@ def test_copyAliasedSender(self):
         """
         exc = self.assertRaises(SystemExit, parseOptions, ['-om'])
         self.assertEqual(exc.args, ("Unsupported option.",))
+
+
+    def test_version(self):
+        """
+        The I{--version} option displays the version and raises
+        L{SystemExit}.
```

Comment:
Maybe: raises  L{SystemExit with `None` as the exit code so.

In this way, we don't need to read this implementation detail as a comment inside the test's implementation.